bin/api-docs/gen-theme-reference.js

```diff
@@ -160,6 +161,23 @@ ${ markupFn( data ) }
 
 let autogen = '';
 
+/**
+ * Format list of types.
+ *
+ * @param {Object} prop
+ * @return {string} type
+ */
+const formatType = ( prop ) => {
+	let type = prop.type || '';
+
+	if ( prop.hasOwnProperty( 'anyOf' ) || prop.hasOwnProperty( 'oneOf' ) ) {
+		const types = prop.anyOf || prop.oneOf;
+		type = types.map( ( item ) => item.type ).join( ', ' );
+	}
```

Comment:
`formatType` is currently adding a trailing comma when extracting types from reference properties as they don't include a `type` property.

```js
[ { type: 'string' }, { '$ref': '#/definitions/refComplete' } ]
/*
  ### filter

CSS and SVG filter styles.

| Property | Type    | Props |
| -------- | ------- | ----- |
| duotone  | string, |       |
*/
```

Filtering out entries that don't include `type` fixes this.

```suggestion
	if ( prop.hasOwnProperty( 'anyOf' ) || prop.hasOwnProperty( 'oneOf' ) ) {
		const types = prop.anyOf || prop.oneOf;
		type = types
			.filter( ( item ) => !! item.type )
			.map( ( item ) => item.type )
			.join( ', ' );
	}
```

We might want to add a way to describe the `$ref` property, but that's probably better left for later.

Reply:
Ah yeah good spot, I guess I was thinking of tackling this when the $ref type had been added. 

$ref will always be an object, so maybe if $ref exists, we could just list 'object'? I've pushed a change that does this to see what you think 🙏

Reply:
If there's no specific type for `$ref`, it should be fine to use `object`